### ISSUE TITLE
训练问题

### ISSUE THREAD

**Issue by zhangshilong2000** [Nnn-s/CaTdiffusion#2]
您好，该项目代码是否支持自定义训练？

**Comment by Nnn-s** [comment]
You can set the number of epochs or the loss function for each stage yourself. The training data for this project's code includes OpenImages V6 and Laion5B, but you can train on other datasets.

**Comment by zhangshilong2000** [comment]
> 您可以自行设置每个阶段的 epoch 数或损失函数。此项目代码的训练数据包括 OpenImages V6 和 Laion5B，但您可以在其他数据集上进行训练。

感谢您的解答，祝您生活愉快！
